src/JavaSession1/ArrayIntersection.java

```diff
@@ -0,0 +1,71 @@
+package JavaSession1;/*Question 5: Find common elements between two arrays.*/
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ArrayIntersection {
+
+    public static void main(String[] args) {
+
+        int[] firstArray;
+        int[] secondArray;
```

Comment:
Should avoid creating blank variables like this. Do it only if it required to do so.
m1() {
  var abc;
   if (expr) {
      abc;
   }
 sout(abc)
}